crates/nu-cmd-extra/src/extra/bits/not.rs

```diff
@@ -119,14 +144,23 @@ impl Command for BitsNot {
                     Span::test_data(),
                 )),
             },
+            Example {
+                description: "Apply logical negation to binary data",
+                example: "0x[ff 00 7f] | bits not",
+                result: Some(Value::binary(vec![0x00, 0xff, 0x80], Span::test_data())),
+            },
         ]
     }
 }
 
-fn operate(value: Value, head: Span, signed: bool, number_size: NumberBytes) -> Value {
-    let span = value.span();
-    match value {
+fn action(input: &Value, args: &Arguments, span: Span) -> Value {
+    let Arguments {
+        signed,
+        number_size,
+    } = *args;
+    match input {
         Value::Int { val, .. } => {
+            let val = *val;
```

Comment:
```suggestion
        Value::Int { &val, .. } => {
```
To avoid that extra assignment

Reply:
for me this does not compile (parse error), and using `ref val` instead does not have the expected effect

Reply:
Ah sorry, never mind then